TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -0,0 +1,21 @@
+@objc public class TradeItSDK: NSObject {
```

Comment:
Can you actually check this branch out and run it. It compiles and runs, but I still get compilation ERRORS about `TradeItSDK` not found in various classes. It's really weird. I cleaned the build completely but I still see the errors in xcode.